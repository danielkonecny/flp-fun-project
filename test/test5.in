A,B,C
a,b
A
A->B
B->C
C->a
C->b
C->AbA