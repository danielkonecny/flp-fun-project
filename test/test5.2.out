A,B,C,<bA>,b'
a,b
A
A->a
A->b
A->A<bA>
<bA>->b'A
b'->b
B->a
B->b
B->A<bA>
C->a
C->b
C->A<bA>